daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -0,0 +1,85 @@
+{{% /alert %}}
+
+Let's say you want to run several applications locally to test them together, similar to a production scenario. With a local Kubernetes cluster, you'd be able to do this with helm/deployment YAML files. You'd also have to build them as containers and set up Kubernetes, which can add some complexity. 
+
+Instead, you simply want to run them as local executables in self-hosted mode.  However, self-hosted mode requires you to:
+
+- Run multiple `dapr run` commands
+- Keep track of all ports opened (you cannot have duplicate ports for different applications).
+- Remember the resources folders and configuration files that each application refers to.
+- Recall all of the additional flags you used to tweak the `dapr run` command behavior (`--app-health-check-path`, `--dapr-grpc-port`, `--unix-domain-socket`, etc.)
+
+With Multi-app Run, you can start multiple applications in self-hosted mode using a single `dapr run -f` command using a template file. The template file describes how to start multiple applications as if you had run many separate CLI `run`commands. By default, this template file is called `dapr.yaml`. 
+
+## MapR template file
+
+When you execute `dapr run -f`, uses the multi-app template file to run all the applications. 
+
+You can name template file with preferred name other than the default. For example `dapr run -f ./<your-preferred-file-name>.yaml`.
+
+The following `dapr.yaml` example includes some of the template properties you can customize for your applications. In the example, you can simultaneously launch 2 applications with app IDs of `processor` and `emit-metrics`.
+
+```yaml
+version: 1
+apps:
+  - appID: processor
+    appDirPath: ../../../apps/processor/
+    appPort: 9081
+    daprHTTPPort: 3510
+    command: ["go","run", "app.go"]
+  - appID: emit-metrics
+    appDirPath: ../../../apps/emit-metrics/
+    daprHTTPPort: 3511
+    env: 
+      DAPR_HOST_ADD: localhost
+    command: ["go","run", "app.go"]
+```
+
+For a more in-depth example and explanation of the template properties, see [Multi-app template]({{< ref multi-app-template.md >}}).
+
+## Locations for resources and configuration files
+
+You have options on where to place your applications' resources and configuration files when using MapR.  
+
+### Point to one file location (with convention)
+
+You can set all of your applications resources and configurations at the `~/.dapr` root. This is helpful when all applications share the same resources path, like when testing on a local machine.
+
+### Separate file locations for each application (with convention)
+
+When developing multiple applications, each app directory can have a `.dapr` folder, which contains a `config.yaml` file and a `resources` directory. Otherwise, if the `.dapr` directory is not present within the app directory, the default `~/.dapr/resources/` and `~/.dapr/config.yaml` locations are used.
```

Comment:
```suggestion
When using MapR, each application directory can have a `.dapr` folder, which contains a `config.yaml` file and a `resources` directory. Otherwise, if the `.dapr` directory is not present within the app directory, the default `~/.dapr/resources/` and `~/.dapr/config.yaml` locations are used.
```